deploy/vagrant/conf/ec2.yml

```diff
@@ -6,12 +6,67 @@ Keypair: "aws-east"
 # Path to the ssh key
 Key_Path: "~/.ssh/aws-east.pem"
 
+# Differences among EC2 instance types: http://aws.amazon.com/ec2/instance-types/
+# When using spark, we recommend "r3.large", compile spark needs 2G memory at least
+#
+# Some Instance Type must need PV/HVM(refer to AMI below)
+Instance_Type: "m1.small"
+
+# tag to the instance name to avoid name conflict during mutiple deployment
+# specify this to illustrate the usage of your cluster, especially when you
+# share the EC2 with others, without tags, it's hard to tell which cluster is yours. 
+#
+# The machines will be named to {Tag}TachyonMaster, {Tag}TachyonWorker1...
+Tag: "my-"
+
+# By default, EC2 instance only mounts 10GB under root directory '/'
+# Different instance types provide different amount of extra disk spaces,
+# but if you do not specify these extra disks in block device mapping, they won't be used.
+# Also, you can specify EBS disks here
+#
+# For more info:
+#   http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/block-device-mapping-concepts.html
+#
+# If you need to specify this field, don't forget to delete the "[]"
+# The devices specified here will be automatically formatted to ext4 and
+# mounted to "/disk0", "/disk1"... in sequence
+#
+# If ufs is HDFS, all the /disk* will be used as HDFS's storage
+# Otherwise, all the /disk* will be used as Tachyon's HDD Tiered Storage
+#
+# Example: 
+# 1. Instance Extra Storage(If the Storage colume specified in instance type matrix
+#    http://aws.amazon.com/ec2/instance-types/ is not "EBS Only")
+# 2. EBS Storage
+Block_Device_Mapping: []
+#  - DeviceName: "/dev/sdz" # EBS Storage
+#    VirtualName: "ebs1"
+#    Ebs.VolumeSize: "100" #GB
+#    Ebs.DeleteOnTermination: "true"
+#  - DeviceName: "/dev/sdf" # Instance Extra Storage
+#    VirtualName: "ephemeral0" 
+#  - DeviceName: "/dev/sdg"
+#    VirtualName: "ephemeral1"
+#    # for 2 x 16 SSD, one is "ephemeral0", another is "ephemeral1", 
+#    # for more, increase the count to "ephemeral2"...
+
+############################################################################
+# Advanced Configurations
+#
+# default values will just work out of box
+# modify only when you know what you are doing
+#
+
 # RHEL 6.5 Amazon Machine Images
 # "PV" and "HVM" are different virtualization methods:
 #   http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/virtualization_types.html
+#
 # The provided AMIs below are only available in east region, for other regions, 
 # click on "Launch Instance" on your EC2 web console, then a list of available AMIs is shown,
 # and you can find the AMI you want.
+#
+# Some Instance Type must need PV/HVM:
+#   http://aws.amazon.com/amazon-linux-ami/instance-type-matrix/
```

Comment:
m1.* is last generation instances, I will change the default instance to "t2.small"